Fix/bclink formatting version 5 3 1

calmacx opened the pull request.

spco commented on coconnect/cdm/objects/common.py:
```diff
@@ -25,17 +25,10 @@ class BadInputs(Exception):
-        self['DATETIME'] = lambda x : pd.to_datetime(x,errors='coerce').dt.strftime('%Y-%m-%d %H:%M:%S')
-        self['DATE'] = lambda x : pd.to_datetime(x,errors='coerce').dt.date
+        self['Integer'] = lambda x : pd.to_numeric(x,errors='coerce').astype('Int64')
+        self['Text50'] = lambda x : x.fillna('').astype(str).apply(lambda x: x[:50])
```
Missing `Text20`

spco reviewed (commented):
Looks good - can we also remove https://github.com/CO-CONNECT/co-connect-tools/blob/master/coconnect/data/cdm/OMOP_CDM_v5_3_1.csv if that's been superceded?

calmacx commented:
* CI fixed, have updated the test data (closing #69)
* removed the old v5.3.1 
* implemented a fix/catch on the command `coconnect display diff` to raise an error for differing columns
